lib/themes/dosomething/paraneue_dosomething/includes/form.inc

```diff
@@ -205,33 +205,23 @@ function paraneue_dosomething_form_dosomething_payment_form_alter(&$form, &$form
 function paraneue_dosomething_form_user_profile_form_alter(&$form, &$form_state, $form_id) {
   $is_resetting_password = $form['account']['current_pass']['#access'] === false;
 
-  // If user was created by Northstar & isn't resetting their password, redirect them to edit profile there.
-  if (module_exists('dosomething_northstar') && dosomething_user_get_field('field_created_by_openid_connect') && !$is_resetting_password) {
+  // Unless the user is resetting their password, send them to Northstar's profile form.
```

Comment:
Oh, so we don't want them going to profile.dosomething.org when resetting passwords?